website/blog/2023-04-19-running-a-local-kubernetes-cluster-with-podman-desktop.md

```diff
@@ -0,0 +1,243 @@
+1. Installing and initializing your container engine: Podman.
+1. Installing and starting your local Kubernetes provider: Kind.
+1. Starting the Redis leader.
+1. Starting and scaling the Redis followers.
+1. Starting and exposing the Guestbook frontend.
+
+<!--truncate-->
+
+## Installing Podman Desktop
+
+You need Podman Desktop.
+
+1. Go to [Podman Desktop installation documentation](/docs/Installation).
+1. Click on your platform name: [Windows](/docs/installation/windows-install), [macOS](/docs/installation/macos-install), or [Linux](/docs/installation/linux-install).
+1. Follow the instructions. Stick to the default installation method.
+1. Start **Podman Desktop**.
+
+At this point, you have a graphical user interface to:
+
+- Install Podman and Kind.
+- Control and work with your container engines and Kubernetes clusters.
+- Run your application on your container engine and migrate it to Kubernetes.
+
+## Installing and initializing your container engine: Podman
+
+Podman Desktop can control various container engines, such as:
+
+- Docker
+- Lima
+- Podman
+
+Consider installing the Podman container engine for:
+
+- Added security
+- No daemon
+- Open source
+
+Containers are a Linux technology.
+
+- On Linux, you can install Podman natively.
+  See: [Installing Podman on Linux](https://podman.io/getting-started/installation#installing-on-linux).
+- On macOS and Windows, Podman requires to run in a Linux virtual machine: the Podman machine.
+  Use Podman Desktop to install Podman and initialize your Podman machine:
+
+#### Procedure
+
+1. Open Podman Desktop **Dashboard**
+1. The **Dashboard** displays _Podman Desktop was not able to find an installation of Podman_.
+1. Click on **Install**.
+1. Podman Desktop checks the prerequisites to install Podman Engine. When necessary, follow the instructions to install prerequisites.
+1. Podman displays the dialog: _Podman is not installed on this system, would you like to install Podman?_. Click on **Yes** to install Podman.
+1. Click on **Initialize and start**.
+
+#### Verification
+
+- The **Dashboard** displays _Podman is running_.
+
+  ![Podman is running](img/running-a-local-kubernetes-cluster-with-podman-desktop/podman-is-running.png)
+
+At this point, you can start working with containers.
+
+## Installing and starting your local Kubernetes provider: Kind
+
+You want to deploy your application to a local Kubernetes cluster.
+
+Podman Desktop can help you run Kind-powered local Kubernetes clusters on a container engine, such as Podman.
+
+Podman Desktop helps you [installing the `kind` CLI](/docs/kubernetes/kind/installing-kind):
+
+1. In the status bar, click on **Kind**, and follow the prompts.
+1. When the `kind` CLI is available, the status bar does not display **Kind**.
+1. On Windows, [configure Podman in rootful mode](docs/kubernetes/kind/configuring-podman-for-kind-on-windows)
+
+   ```shell-session
+   $ podman system connection default podman-machine-default-root
```

Comment:
I am testing in Windows and I am not able to properly run podman nor kind... This is not problem of the this nice blog post, though. 
On linux... I am seeing multiple pods being run but with some problems: `too many files open` inside pods. 
I will try to debug the issues and report them, in a meantime this could be finished.

Reply:
ack, not a blocker to publish